bvm/ballerina-runtime/src/main/java/module-info.java

```diff
@@ -63,6 +63,7 @@
     exports io.ballerina.runtime.internal.values to io.ballerina.testerina.core, io.ballerina.testerina.runtime,
             io.ballerina.lang.xml, org.ballerinalang.debugadapter.runtime;
     exports io.ballerina.runtime.internal.configurable to io.ballerina.lang.internal;
+    exports io.ballerina.runtime.internal.configurable.providers.toml to org.ballerinalang.debugadapter.runtime;
```

Comment:
It seems, here we need the toml-specific class `TomlDetails` which contains the config file info to call the `$configureInit` method. Do we need to move the class to a more common place? @warunalakshitha

Reply:
I think we already expose internal packages to debugger runtime hence this should be ok.

Reply:
Yeah all the runtime internals are exposed only to debugger runtime(not to the debug server core).